nautobot_chatops_ansible/tests/test_tower.py

```diff
@@ -115,7 +115,7 @@ def test_get_tower_inventory_id(self):
             test_tower = Tower(
                 origin=SLACK_ORIGIN, tower_uri="https://mocktower", username="mock", password="mock", verify_ssl=False
             )
-            inventory_id = test_tower.get_tower_inventory_id(inventory_name="Nautobot SoT Agg Inventory")
+            inventory_id = test_tower.get_tower_inventory_id(inventory_name="NautobotSOT")
```

Comment:
I think this looks good, but a suggestion is to move "NautobotSOT" to fixtures as a dedicated variable, and just reference the var from there instead.  Same for line 131

Reply:
I'm open to this suggestion, but am not following.